cornedbeef/build.gradle

```diff
@@ -24,3 +29,42 @@ dependencies {
     testCompile 'org.robolectric:robolectric:3.0-rc3'
     testCompile 'org.mockito:mockito-all:2.0.2-beta'
 }
+
+publishing {
+    publications {
+        aar(MavenPublication) {
+            groupId "TouchType"
+            version = android.defaultConfig.versionName
+            artifactId project.getName()
+
+            // Tell maven to prepare the generated "*.aar" file for publishing
+            artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")
+        }
+    }
+}
+
+artifactory {
+
+    // artifactory properties with local defaults
+    def user=project.properties['artifactory_user'] ?: "admin"
```

Comment:
Interested in your thoughts on this @eartle ?

The intention is that this publish job will overwrite the params using -Partifactory_user -Partifactory_pass etc.

but is there a better / more standard way of doing this?

I can't reference the properties directly because it will throw an error when they are not specified (even if I'm not publishing)